boards/intel/adsp/twister.yaml

```diff
@@ -0,0 +1,31 @@
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+  - xcc
+  - xt-clang
```

Comment:
twister.yaml data will apply to all variants and boards unless specified otherwise, no need to add each variant and target if they share the same data.

Reply:
#80564